### ISSUE TITLE
calderawp\calderaforms\pro\container not found

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#2204]
On WordPress.org, user reports this error:
`Fatal error: Class ‘calderawp\calderaforms\pro\settings\container’ not found in /home/content/42/7612342/html/wp-content/plugins/caldera-forms/includes/cf-pro-client/classes/settings/active.php on line 21`

https://wordpress.org/support/topic/fatal-error-in-active-php/#post-9838416

This makes no sense. Why? L21 is in a method in the `active` class that is called once. Where? In the class `hooks`, which is instantiated once. Where? In the class `container`.

In summary, `container` calls `hooks` which calls `active` which calls `container` which is now unknown? 

There must be some version of PHP where this:

```
<?php

namespace calderawp\calderaforms\pro\settings;
use calderawp\calderaforms\pro\container;

```

Doesn't make `container()` a valid way to address the class `calderawp\calderaforms\pro\container`

**Comment by Shelob9** [comment]
I'm closing this as it appears to have been a bad download from dot org.
